### PR TITLE
implemented task solution

### PR DESCRIPTION
**Pull request by YevhenKharko**
[DEMO LINK](https://yevhenkharko.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

**Review by sviatikkondur — APPROVED** [review]
Good job!

**Review by sviatikkondur — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { Good } from './components/Good/Good';
+
+enum SortBy {
+  default = 'default',
+  name = 'name',
+  length = 'length',
+}
```
Comment (StasSokolov) [inline src/App.tsx:12]: let's write enum keys with camel case or uppercase https://www.typescriptlang.org/docs/handbook/enums.html

File: src/App.tsx [hunk 2]
```diff
@@ -15,48 +23,92 @@ export const goodsFromServer = [
...
 export const App: React.FC = () => {
+  const [sortBy, setSortBy] = useState(SortBy.default);
+  const [isReversed, setIsReversed] = useState(false);
+
+  const reset = () => {
+    setSortBy(SortBy.default);
+    setIsReversed(false);
+  };
+
+  const goods = getPreparedGoods(goodsFromServer, sortBy, isReversed);
```
Comment (StasSokolov) [inline src/App.tsx:65]: move to the top after `useState`

File: src/App.tsx [hunk 3]
```diff
@@ -15,48 +23,92 @@ export const goodsFromServer = [
...
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
+        {(sortBy !== SortBy.default || isReversed) && (
+          <button
+            type="button"
+            className="button is-danger is-light"
+            onClick={() => reset()}
```
Comment (StasSokolov) [inline src/App.tsx:103]:
```suggestion
            onClick={reset}
```
can be more simplify

**Comment by YevhenKharko** [comment]
Sorry for closing PR, deleted repository by accident, so must create new PR

**Pull request closed without merging**